cdc/model/sink.go

```diff
@@ -604,7 +604,10 @@ func BuildTiDBTableInfo(columns []*Column, indexColumns [][]int) *model.TableInf
 			continue
 		}
 		if firstCol.Flag.IsPrimaryKey() {
```

Comment:
maybe we can postpone the recover of PK in line 617's loop

Reply:
Thank you for your advice!